src/utils/mtev_lfu.c

```diff
@@ -236,6 +263,12 @@ touch_lfu_cache_no_lock(mtev_lfu_t *c, struct lfu_entry *e)
 mtev_boolean
 mtev_lfu_put(mtev_lfu_t *lfu, const char *key, size_t key_len, void *val)
 {
+  /* a max size of zero means to disable the LFU */
+  if (lfu->max_entries == 0) {
+    /* we don't want calling code to error */
+    return mtev_true;
```

Comment:
Why don't you want calling code to error, here? The way this is written, this signals success, but `mtev_lfu_get` and `mtev_lfu_remove` will not return a valid value...